TreePiece.cpp

```diff
@@ -1983,6 +1995,11 @@ void TreePiece::assignDomain(const CkCallback &cb)
     contribute(cb);
     }
 
+inline double SHEAR(int ix, double t, Vector3D<cosmoType> fPeriod, double dOrbFreq, GravityParticle *p) {
+    ((ix) < 0 ? fmod(0.5 * fPeriod[1] - 1.5 * (ix) * dOrbFreq * fPeriod[0] * (t), fPeriod[1]) - 0.5 * fPeriod[1] : \
```

Comment:
Clean this up by removing unnecessary parentheses, the final backslash, and change it to a more readable if-else if-else statement.